src/assets/toolkit/styles/utils/text.css

```diff
@@ -16,6 +16,16 @@
   font-weight: var(--font-weight-semibold) !important;
 }
 
+
+/**
+ * Text color
+ */
+
+.u-textWhite {
+  color: var(--color-white);
+}
+
```

Comment:
Okay, I removed that change from this PR and will address contrast in another one.